packages/app/src/navigation/SidebarNavigation.vue

```diff
@@ -178,16 +184,22 @@ const debugBadge = computed<Badge | undefined>(() => {
     }
 
     if (testsForReview?.length) {
+      const valueToDisplay = testsForReview.length < 9
+        ? String(testsForReview.length)
+        : '9+'
+
+      if (status === 'ERRORED') {
+        return { value: valueToDisplay, status: 'error', label: t('sidebar.debug.errored') }
+      }
+
       return {
-        value: testsForReview.length < 9
-          ? String(testsForReview.length)
-          : '9+',
+        value: valueToDisplay,
         status: 'failed',
         label: t('sidebar.debug.failed', testsForReview.length),
       }
     }
 
-    return { value: '0', status: 'error', label: t('sidebar.debug.errored') }
```

Comment:
This was to cover any error status like 'OVERLIMIT' and others. Are we not supposed to show the badge for these cases?

Reply:
I rearranged to make sure those cases were covered.